tools/create-image.sh

```diff
@@ -26,6 +27,7 @@ display_help() {
     echo
     echo "   -d, --distribution         Set on which debian distribution to create"
     echo "   -f, --feature              Check what packages to install in the image, options are minimal, full"
+    echo "   -s, --seek                 Set img size, default 2047(2G)"
```

Comment:
It's a bit weird that we expose the internal detail (dd seek) in public-facing flags.
I would rename it to "size", mention the units (MB).
I am not sure how good bash at math, but is it possible to accept full size (2048) from user and then subtract 1?

Reply:
Yes，I agree to subtract 1

Reply:
Please ping this PR when you do the change then.
Thanks!

Reply:
It's my honor. Thanks for you to create this great project.